cocoa/core/dataset.py

```diff
@@ -114,10 +114,10 @@ def read_examples(paths, max_examples, Scenario):
     examples = []
     for path in paths:
         print 'read_examples: %s' % path
-        for raw in read_json(path):
+        for example in read_json(path):
             if max_examples >= 0 and len(examples) >= max_examples:
                 break
-            examples.append(Example.from_dict(raw, Scenario))
+            examples.append(Example.from_dict(example, Scenario))
     return examples
```

Comment:
hmm. not sure why you changed raw to example. but raw is more informative here. example is like the Example object.

Reply:
changed to match the unit_test below, i guess the better idea would have been to change the unit_test to match the actual code.  i will do that now